classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    students.Classroom = self
+    @students << student
+  end
```

Comment:
- [ ] @batoorsohail great work adding  a method that serves as the 'has-many' side of the association, however, you forgot to add ` a method that serves as the 'belongs-to' side association` in your student class, kindly consider adding it to make the association complete.